src/schema/v2/sale/index.ts

```diff
@@ -200,6 +201,7 @@ export const SaleType = new GraphQLObjectType<any, ResolverContext>({
           eligible_sale_artworks_count,
       },
       endAt: date,
+      endedAt: date,
```

Comment:
It's fine to add this, but there are other ways to tell if a sale is closed on the client (for instance `isClosed`).